### PR TITLE
Use Rollup.js to create UMD package so that this can be used in the browser

### PR DESCRIPTION
**Pull request by bartboy011**
Currently this package can only be used in Node.js projects, even with a build pack (due to the dynamic require statements). It would be much more useful if this package could be used in all JS environments. This PR introduces Rollup.js as a buildpack to create a UMD package that can be used in all environments, including the browser as a drop-in file. 

Additionally, I've added Babel to the build system so that ES6 can be used without concern.

Finally, I've removed the examples folder as it was redundant and added unnecessary weight to installs.

@ActiveCampaign/integration

### CODE REVIEW

**Comment by bartboy011** [comment]
After doing some browser testing I discovered that there were some issues in the build. I retooled the build and that exposed issues with the `request` library. I replaced `request` with `axios`, this finally got everything working, at which point I discovered that we don't set CORS headers... This package is now in great shape, but without those headers in our API it is still unusable in the browser.

See this for clarity: http://screen.ac/3S460k071A2l

File: lib/Connector.js [hunk 1]
```diff
@@ -133,15 +130,15 @@ var Connector = {
 				body = body.substring(0, body.length - 1);
 			}
 		}
-		
+
 		var request_options = {
 			method: this.request_method,
 			url: url,
-			body: body
+			data: body
```
Comment (mthommes) [inline lib/Connector.js:137]: If `body` includes `dataType: "jsonp"`, that might resolve the CORS issue.

Comment (bartboy011) [reply lib/Connector.js:137]: @mthommes I had to replace `request` with `axios` as `request` had some issues with being bundled, and `axios` does not support jsonp.

**Comment by bartboy011** [comment]
@cristiangrama This is ready for a review + merge!

File: build/dev.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+var colors = require('colors');
```
Comment (cristiangrama) [inline build/dev.js]: this is not used anywhere.

File: lib/Connector.js [hunk 3]
```diff
@@ -151,44 +148,33 @@ var Connector = {
...
-		var api_request = new RSVP.Promise(function(fulfill, reject) {
-			request(request_options, function(error, response, body) {
+		var api_request = axios(request_options)
+			.then((response) => {
 				try {
-					var result = JSON.parse(body);
+					JSON.parse(body);
 				} catch (e) {
-					apiWhitelist.forEach(function(value) {
+					apiWhitelist.forEach(function (value) {
```
Comment (cristiangrama) [inline lib/Connector.js:156]: I thought we have removed that already. Is that still supposed to be there?

**Review by cristiangrama — CHANGES_REQUESTED** [review]

File: build/dev.js (hunk 2 above)
Comment (bartboy011) [reply build/dev.js]: It's a weird package, requiring it adds prototype methods to the `String` class, such as the `.black.bgCyan` on line 12.

File: lib/Connector.js (hunk 3 above)
Comment (bartboy011) [reply lib/Connector.js:156]: Yeah, I believe this whitelist works in a different way than the other class-specific whitelists. I don't fully understand how it's used yet so I'm not confident in removing it. Maybe future exploration will reveal that it is no longer necessary.

File: build/dev.js (hunk 2 above)
Comment (bartboy011) [reply build/dev.js]: I think I can probably not assign the return value to a const though, I'll make a revision here.

**Comment by bartboy011** [comment]
@cristiangrama Updated!

**Review by cristiangrama — APPROVED** [review]

**Pull request merged**